Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3431,6 +3433,25 @@ - (void)updateLastVisibleTimestamp
     self.hasUnreadMessages = numberOfUnreadMessages > 0;
 }
 
+- (void)cleanUpUnreadIndicatorIfNecessary
+{
+    BOOL hasUnreadIndicator = self.dynamicInteractions.unreadIndicatorPosition != nil;
```

Comment:
If there is an unread indicator and we can't find any unread messages, use `ensureDynamicInteractions` to clean up the obsolete unread indicator.